content/en/docs/concepts/architecture/controller.md

```diff
@@ -96,6 +96,17 @@ the current state closer in line.
 nodes in your cluster. See
 [Cluster autoscaling](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaling)).
 
+If you wanted to, you could [extend Kubernetes](/docs/concepts/extend-kubernetes/)
```

Comment:
@sftim. Here is some feedback:
nit: This paragraph is written like a conversation with an audience. I suggest you remove
the conversational bits since this is a concept page. Specifically:
Why would you do that? Well, you probably wouldn't - you could just buy a thermostat.
In the world of IT
Suggested edit:
nit: In the thermostat example, if the room is very cold then a different controller
might also turn on a frost protection heater. With Kubernetes clusters, your control plane interfaces
with IP address management tools, APIs to provision and configure
load balancers, storage services, and possibly other services.

Reply:
Thanks for the feedback. Revised.